src/ui/BtnAdd/component.tsx

```diff
@@ -0,0 +1,7 @@
+const BtnAdd = () => {
+    return (
+      <button type="button">Enviar</button>
+    )
+  }
+
+  export default BtnAdd
```

Comment:
pleas fix the indentation (remove the space at the beginning of the line), and also add a final `enter` at the end of the file

and do the same for all other files

Reply:
ready!!!